docker-compose.yml

```diff
@@ -51,15 +51,14 @@ nfs:
     - 2049
   labels:
     - triton.cns.services=nfs
-    # Soft anti-affinity to avoid all other containers
```

Comment:
The reason was that the comments existed in the first block only, with that one exception. Should I put the comments in everywehre?